### PR TITLE
RHMAP-2926 - Unable to tail logs using FHC

### PR DESCRIPTION
**Pull request by wtrocki**
# What was done:
* Moved depreciated implementation to support only old version of API
File lib/cmd/common/app/logs.js is now in lib/cmd/fh2/app/logs.js 
and will work only fort FH2. No changes there because this version of APi is depreciated.
* Created dedicated version for FH3 using new command line format.
* Reviewed *log* api to check for possible problems/vulnerabilities.
* Reviewed documentation for possible updates.
* Implementation is trivial. No unit tests created.

Please note that format of arguments is different than before.

### CODE REVIEW

**Comment by fheng-build-servers** [comment]
@maleck13 @jasonmadigan please review this PR.

File: lib/cmd/fh3/app/logs/get.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+var fhc = require("../../../../fhc");
+
+module.exports = {
+  'desc': 'Fetch log file contents from specified environment.',
+  'examples': [{
+    cmd: 'fhc app get --app=2bf4gg3gfefsdgfdsg4fdgs34 --env=dev --logname=env.dev.log',
```
Comment (wei-lee) [inline lib/cmd/fh3/app/logs/get.js]: fhc app logs get

File: lib/cmd/fh3/app/logs/list.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+var fhc = require("../../../../fhc");
+var ini = require('../../../../utils/ini');
+var common = require("../../../../common");
+var Table = require('cli-table');
+var moment = require('moment');
+
+module.exports = {
+  'desc': 'Get list of application log files',
+  'examples': [{
+    cmd: 'fhc app list --app=2bf4gg3gfefsdgfdsg4fdgs34 --env=dev',
```
Comment (wei-lee) [inline lib/cmd/fh3/app/logs/list.js]: fhc app logs list

**Comment by wei-lee** [comment]
Small issues about the description. Have tried locally and works as expected.

**Comment by wtrocki** [comment]
Fixed

**Comment by david-martin** [comment]
@wtrocki I'm getting a similar error that was originally seen for 'fhc logs' cmd, but for 'fhc runtimes'

```fhc runtimes --env=qed1-mbaas2-env
fhc ERR! Error: Error - not 2xx status code. (500)
fhc ERR! {"date":"Thu Apr 13 09:55:19 UTC 2017 - 1492077319104","msg":"host parameter is null","status":"error"}
fhc ERR! Request ID:  68820365-1fa6-40d2-99f9-5e9454f354f6
fhc ERR! 
fhc ERR! System Linux 4.10.8-200.fc25.x86_64
fhc ERR! command "/usr/bin/node" "/usr/src/app/bin/fhc.js" "runtimes" "--env=qed1-mbaas2-env"
fhc Command executed with error
```
What was the fix in this PR for logs to prevent that error? (Its difficult to see as the file was completely rewritten)

**Comment by wtrocki** [comment]
@david-martin I just removed need for passing app host, by using different api. This error is happening for old commands that are still using millicore api.

**Pull request merged**